docs/ci-integration/guides/gh-actions-integration.md

```diff
@@ -1,6 +1,11 @@
 
 # GitHub Actions integration
 
+{% hint style='info' %}
+##### News
+The recommended approach for GHA integration now is [using Satellites as GHA runners](../cloud/satellites/gha-runners.md).
```

Comment:
```suggestion
It is recommended to use [Satellites as GHA runners](../cloud/satellites/gha-runners.md), since they don't require github actions workers.
```